lib/SILOptimizer/SILCombiner/SILCombinerMiscVisitors.cpp

```diff
@@ -745,32 +745,6 @@ SILInstruction *SILCombiner::visitAllocStackInst(AllocStackInst *AS) {
   return eraseInstFromFunction(*AS);
 }
 
-SILInstruction *SILCombiner::visitAllocRefInst(AllocRefInst *AR) {
```

Comment:
Nice! Good to remove things from SILCombine like this!

I am assuming that you validated that all of the test cases in the SIL below are also in DeadObjectElimination. If not, can you check that and add any that are missing!

Reply:
yes, all the cases are tested in dead_alloc_elim.sil and dead_alloc_elim_ossa.sil